### ISSUE TITLE
no-implicit-dependencies

### ISSUE THREAD

**Issue by AviVahl** [wix/tslint-wix-react#9]
tslint 5.8.0 now has **no-implicit-dependencies** turned on by default.
The rule verifies import statements only refer to packages that are listed as "dependencies" in package.json.

Meaning, if I:
```ts
import {Component} from 'react';
```
It throws an error if react isn't listed as a dependency.

Although it sounds great at first glance, it does have issues:
- Ignores peerDeps, which really sucks for our libraries, as we don't want them to force a specific react/stylable version.
- Cannot work in our current tests setup, as we use "devDependencies" packages in our tests (chai, sinon, etc.)

I vote to turn it off in our preset for now.

Thoughts?

**Comment by tyv** [comment]
I agree

**Comment by tyv** [comment]
I think you may proceed

**Comment by AviVahl** [comment]
@tyv could you release a new version with this rule included?
```
npm version patch
npm publish
git push
git push --tags
```

**Comment by tyv** [comment]
@AviVahl we don't release patches. 
https://github.com/wix/tslint-wix-react/blob/master/.github/CONTRIBUTING.md

I will do, thanks!

**Comment by tyv** [comment]
@AviVahl published as `5.0.0`
